ui/app/components/alert-popup.js

```diff
@@ -0,0 +1,7 @@
+import Component from '@ember/component';
+
+export default Component.extend({
```

Comment:
I think this will be a good candidate for a template-only component so we should probably extend the outer-html component here (so there's no additional wrapping div).